src/lib/mina_metrics/prometheus_metrics/mina_metrics.ml

```diff
@@ -344,13 +343,20 @@ module Transaction_pool = struct
     in
     Counter.v "transactions_added_to_pool" ~help ~namespace ~subsystem
 
-  let parties_transaction_size : Gauge.t =
+  let zkapp_transaction_size : Gauge.t =
     let help = "Size of valid parties transaction received (bin_size_t)" in
-    Gauge.v "parties_transaction_size" ~help ~namespace ~subsystem
+    Gauge.v "zkapp_transaction_size" ~help ~namespace ~subsystem
 
-  let parties_count : Gauge.t =
+  let zkapp_updates : Gauge.t =
```

Comment:
Likely ditto for the updates (counter for # of zkapps transactions can be shared across these metrics though).